src/kagglehub/auth.py

```diff
@@ -14,3 +17,19 @@ def login():
     set_kaggle_credentials(username=username, api_key=api_key)
 
     logger.info("Kaggle credentials set.")
+
+    if validate_credentials is False:
```

Comment:
nit: you can also use `if not validate_credentials:`. It is a bit more pytonic I think.